draft-ietf-rtcweb-jsep.xml

```diff
@@ -2195,13 +2195,16 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           <xref target="I-D.ietf-mmusic-sctp-sdp"></xref>, Section
           4.1.</t>
 
-          <t>Within the data m= section, the "a=mid", "a=ice-ufrag",
+          <t>Within the data m= section, an "a=mid" line MUST be
+          generated and included as described above,
+          along with an "a=fmtp:webrtc-datachannel" line and an
+          "a=sctp-port" line referencing the SCTP port number as defined in
+          <xref target="I-D.ietf-mmusic-sctp-sdp"></xref>, Section 4.1.
+          If the m= section either has a unique address or is associated
```

Comment:
Make a new t section here, and start with "As discussed above, the following attributes are included only if ..."